topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.md

```diff
@@ -0,0 +1,514 @@
+3. Circular replicons are 'rotated' (have their starting position shifted) between rounds of polishing to ensure that **no part of the sequence is left unpolished**.
+
+
+Let's try it on our data!
+
+> ### {% icon hands_on %} Hands-on: Unicycler assembly
+>
+> 1. **Create assemblies with Unicycler** {% icon tool %} with the following parameters
+>   - *"Paired or Single end data"*: `None`
+>   - *"Select long reads. If there are no long reads, leave this empty"*: The `Plasmids` dataset collection
+>
+> 2. **Bandage image** {% icon tool %} with the following parameters
+>   - *"Graphical Fragment Assembly"*: the `Final Assembly Graph` collection created by **Unicycler** {% icon tool %}
+>
+> 3. Examine {% icon galaxy-eye %} the output images again
+>
+> > ### {% icon question %} Question
+> >
+> > For which samples has the plasmid assembly improved?
+> >
+> > > ### {% icon solution %} Solution
+> > > Exploring the outputs for all the samples reveals that many now display circular assemblies, indicating the full plasmids sequence was resolved.
+> > {: .solution }
+> {: .question}
+>
+{: .hands_on}
+
+The Assembly graph image of the RB01 assembly with **miniasm** {% icon tool %} shows one unclear hypothetical plasmid, where the output of **Unicycler** {% icon tool %} shows two clear plasmids, as also shown by {% cite LiXie2018 %}.
+
+![Bandage output](../../images/plasmid-metagenomics-nanopore/Bandage_output.png)
+
+# Species and plasmids
+
+## Prediction of plasmid sequences and classes using PlasFlow
+
+To automatically determine whether the contigs represent chromosomal or plasmid DNA, **PlasFlow** {% icon tool %} ({% cite Krawczyk2018 %}) can be used, also in the case where a full circular plasmid sequence was not assembled. Furthermore, it assigns the contigs to a bacterial class.
+
+**PlasFlow** {% icon tool %} is a set of scripts used for prediction of plasmid sequences in metagenomic contigs.
+It relies on the neural network models trained on full genome and plasmid sequences and is able to differentiate between plasmids and chromosomes with accuracy reaching 96%.
+
+![Pairwise alignment](../../images/plasmid-metagenomics-nanopore/PlasFlow.png)
+
+
+> ### {% icon hands_on %} Hands-on: Prediction of plasmid sequences
+>
+> 1. **PlasFlow** {% icon tool %} with the following parameters
+>   - *"Sequence Reads"*: the `Final Assembly` collection created by **Unicycler** {% icon tool %}
+>
+> > ### {% icon question %} Question
+> >
+> > What is the classification of contig_id 0 in RB10? (Hint: Check the probability table created by PlasFlow)
+> >
+> > > ### {% icon solution %} Solution
+> > > plasmid.Proteobacteria
+> > >
+> > > This can be determined by looking at the 5th column of the probability table.
+> > {: .solution }
+> {: .question}
+{: .hands_on}
+
+
+The most important output of **PlasFlow** {% icon tool %} is a tabular file containing all predictions (specified with `--output` option), consisting of several columns including:
```

Comment:
Not sure if `(specified with `--output` option)` is useful here.

Reply:
```suggestion
The most important output of **PlasFlow** {% icon tool %} is a tabular file containing all predictions, consisting of several columns including:
```